Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -247,7 +247,12 @@ final class ProjectEditorMapper: ProjectEditorMapping {
             name: projectName,
             organizationName: nil,
             developmentRegion: nil,
-            options: [],
+            options: .init(
+                automaticSchemesOptions: .disabled,
+                disableBundleAccessors: true,
+                disableSynthesizedResourceAccessors: true,
+                textSettings: .init(usesTabs: nil, indentWidth: nil, tabWidth: nil, wrapsLines: nil)
+            ),
```

Comment:
Is this ok (same below)? I don't think we need individual schemes for `tuist edit`, right?

Reply:
Personally, I found it useful when there was another issue that made the whole "edit" project fail to compile, so I could compile a specific target. If there are no benefits in disabling it, I would keep it for now (yet I don't mind removing either).

Reply:
I think 99.9% of the time you just care of compiling everything, and having less schemes is less confusing.
Then, if you want to manually add the scheme for a specific test, you can still do it from Xcode, isn't it?

Reply:
Sounds good.